build.gradle

```diff
@@ -18,6 +18,13 @@ repositories {
 
 allprojects {
     version = '2.7.0'
+
+    // Force spotless and checkstyle depending on newer version of guava due to CVE-2023-2976. Remove after they both upgrades.
+    configurations.all {
+        resolutionStrategy {
+            force "com.google.guava:guava:32.1.2-jre"
```

Comment:
Can we move this down to the same section for resolution?

Reply:
Firstly I tried that, but not only `spotless` depends on old version of `guava`, but also `checkstyle`. `checkstyle` Gradle plugin is applied to the whole project, only set the `resolutionStrategy` for the sub-project is not enough.
There is another way to set version of `checkstyle`, but setting `guava` version is still needed for `spotless`. I put it in a new commit https://github.com/opensearch-project/opensearch-oci-object-storage/pull/56/commits/2080cafb2ac6c38bafcf9eedb8d7787363df6e59. What do you think of the new commit? If that's not better, I can revert it.

Reply:
I'm testing whether `guava` version needs to be assigned explicitly for `checkstyle` plugin in this commit https://github.com/opensearch-project/opensearch-oci-object-storage/pull/56/commits/b4a4cf2e037182ee64e8e2465fda3a8d9a4669c4, if the PR check of vulnerability fails, I will revert it.